apache/trafficcontrol
# [TPv2] Update page titles and url for create() call

Closes: #7418

<hr/>

## Which Traffic Control components are affected by this PR?

- Traffic Portalv2

## What is the best way to verify this PR?

1. Create a new physical locations/capabilities/coordinates/divisiona/regions/asns/roles/uers/tenants/types/origins/parameters/profiles and it should update the title and route of the page.
2. Update a physical locations/capabilities/coordinates/divisiona/regions/asns/roles/uers/tenants/types/origins/parameters/profiles etc and it should update the title and NOT the route of the page.

## If this is a bugfix, which Traffic Control versions contained the bug?

master

## PR submission checklist
